financial-templates-lib/Logger.js

```diff
@@ -33,11 +35,38 @@ const transports = [
   })
 ];
 
+if (process.env.SLACK_WEBHOOK) {
+  transports.push(
+    new SlackHook({
+      webhookUrl: process.env.SLACK_WEBHOOK,
+      formatter: info => {
+        return {
+          text: `${info.level}: ${info.message}`,
+          attachments: [
+            {
+              text: `${JSON.stringify(info)}`
+            }
+          ],
+          blocks: [
+            {
+              type: "section",
+              text: {
+                type: "plain_text",
+                text: `${info.level}`
+              }
+            }
+          ]
+        };
+      }
+    })
+  );
+}
+
 transports.push(
   new winston.transports.Console({
     level: "debug",
-    handleExceptions: true,
-    format: alignedWithColorsAndTime
+    handleExceptions: true
+    // format: alignedWithColorsAndTime
```

Comment:
Why is this commented? Is it related to an issue? If so, could you add a TODO?

Reply:
This formatter requires more refinement. will add a Todo